model/task_queue.go

```diff
@@ -74,6 +74,9 @@ func LoadTaskQueue(distro string) (TaskQueueAccessor, error) {
 }
 
 func (self *TaskQueue) Length() int {
+	if self == nil {
```

Comment:
Wouldn't it be better to find where we're calling Length on a nil, and fix that?

Reply:
It would. However, from my reading of the code, it's not possible for it to be nil, since we check that case immediately before calling `assignNextAvailableTask`: https://github.com/evergreen-ci/evergreen/blob/master/service/api_task.go#L445. I haven't been able to figure out why the queue is nil.